File: construction/src/main/java/com/construction/ProjectChange.java
package com.construction;
// Бізнес-об'єкт: опис змін в проєкті
public class ProjectChange {
    private String description;
    private String impact;

    public ProjectChange(String description, String impact) {
        this.description = description;
        this.impact = impact;
    }

    public String getDescription() {
        return description;
    }

    public String getImpact() {
        return impact;
    }

    @Override
    public String toString() {
        return "Change[desc='" + description + "', impact='" + impact + "']";
    }
}

File: construction/python/ProjectChange.py
class ProjectChange:
    def __init__(self, description: str, impact: str):
        self._description = description
        self._impact = impact

    def get_description(self) -> str:
        return self._description

    def get_impact(self) -> str:
        return self._impact

    def __str__(self) -> str:
        return f"Change[desc='{self._description}', impact='{self._impact}']"